vanilla/bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ -z "$WORLD_FILENAME" ]; then
+    /terraria-server/TerrariaServer -world $@
+else
+    /terraria-server/TerrariaServer -world $CONFIGPATH/$WORLD_FILENAME $@
```

Comment:
I think the intent is that worlds are volumed in the `/root/.local/share/Terraria/Worlds` path where the executable defaults saving world files.  Is the intent that the configuration data lives side-by-side with the world file on the host machine?  

Making this change would force users through the bootstrapper.  If they do not have a world file how would that work?  They would have to volume in the directory to save the data and then copy the data to where they store their config.

If I am being honest however, I've spent a lot less time maintaining the vanilla version of the server, and am a bit less familiar with its options relative to how TShock operates.

Reply:
You're right, I shouldn't have the "-world" on line 4. I can fix that.

With respect to the CONFIGPATH, this is really meant to sync up with the tshock version (which defaults to the world path - "ENV CONFIGPATH=/root/.local/share/Terraria/Worlds").

I actually moved over from the tshock version to vanilla (due to the most recent Terraria update), so my goal was to sync up with that. Alternatively, I could add another "WORLDPATH" variable if we wanted to separate the world and configuration paths.

The overall goal was to try avoiding the necessity of hardcoding world file paths into the entrypoint / command, but to do so by using an abstraction that might be helpful in the future (by adding a thin wrapper around the vanilla server launcher).